Formula/k/kubectl-rook-ceph.rb

```diff
@@ -0,0 +1,30 @@
+class KubectlRookCeph < Formula
+  desc "Rook plugin for Ceph management"
+  homepage "https://rook.io/"
+  url "https://github.com/rook/kubectl-rook-ceph/archive/refs/tags/v0.8.0.tar.gz"
+  sha256 "b007bc9e971c253da53023f0448f82091d8dc94f8081789cf9b889e86d364a7a"
+  license "Apache-2.0"
+  head "https://github.com/rook/kubectl-rook-ceph.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"kubectl-rook_ceph"), "cmd/main.go"
+  end
+
+  test do
+    output = shell_output("#{bin}/kubectl-rook_ceph --help")
+    assert_match("kubectl rook-ceph provides common management and troubleshooting tools for Ceph.", output)
```

Comment:
version check might be better

Reply:
I agree but it doesn't provide a version command:

```bash
kubectl rook-ceph --help                                                                                                                                                                                                ✔  tardis ○
kubectl rook-ceph provides common management and troubleshooting tools for Ceph.

Usage:
  rook-ceph [command]

Available Commands:
  ceph            call a 'ceph' CLI command with arbitrary args
  completion      Generate the autocompletion script for the specified shell
  debug           Debug a deployment by scaling it down and creating a debug copy. This is supported for mons and OSDs only
  destroy-cluster delete ALL data in the Rook cluster and all Rook CRs
  dr              Calls subcommand health
  health          check health of the cluster and common configuration issues
  help            Help about any command
  mons            Output mon endpoints
  operator        Calls subcommands like `restart`  and `set <key> <value>` to update  rook-ceph-operator-config configmap
  rbd             call a 'rbd' CLI command with arbitrary args
  restore-deleted Restores a CR that was accidentally deleted and is still in terminating state.
  rook            Calls subcommands like `version`, `purge-osd, status` and etc.
  subvolume       manages stale subvolumes

Flags:
      --context string              Kubernetes context to use
  -h, --help                        help for rook-ceph
      --kubeconfig string           kubernetes config path
  -n, --namespace string            Kubernetes namespace where CephCluster is created (default "rook-ceph")
      --operator-namespace string   Kubernetes namespace where rook operator is running

Use "rook-ceph [command] --help" for more information about a command.
```

(And the completion doesn't work properly since it also requires a running Kubernetes cluster)